### PR TITLE
Typedefs

### PR DESCRIPTION
**Pull request by phadej**
Waits for #266 to be merged

### CODE REVIEW

**Comment by phadej** [comment]
Ready for review

File: hs-bindgen/fixtures/simple_structs.th.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ instance Storable CS2
                                   {MkCS2 cS2_a_3
                                          cS2_b_4
                                          cS2_c_5 -> pokeByteOff x_1 0 cS2_a_3 >> (pokeByteOff x_1 32 cS2_b_4 >> pokeByteOff x_1 64 cS2_c_5)}}
-newtype CS2T = MkCS2T {unCS2T :: Void}
+newtype CS2T = MkCS2T {unCS2T :: CStruct'0020S2}
```
Comment (edsko) [inline hs-bindgen/fixtures/simple_structs.th.txt:18]: What is this `CStruct'0020S2`..?

Comment (phadej) [reply hs-bindgen/fixtures/simple_structs.th.txt:18]: mangled `struct S2`

Comment (edsko) [reply hs-bindgen/fixtures/simple_structs.th.txt:18]: Ah, it's a space. Gotcha.

**Pull request merged**